From 150fcad08402632fe201c8f90bb8669dcc9e0915 Mon Sep 17 00:00:00 2001
From: Andrew Scott <andj.scott@gmail.com>
Date: Mon, 14 Mar 2016 01:07:20 +0000
Subject: [PATCH] Fix buffer overflow in sprintf

---
 src/mbpfan.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mbpfan.c b/src/mbpfan.c
index 17dd759..f369c39 100644
--- a/src/mbpfan.c
+++ b/src/mbpfan.c
@@ -129,7 +129,7 @@ t_sensors *retrieve_sensors()
         int counter;
         for (counter = 0; counter < 10; counter++) {
 
-            char hwmon_path[strlen(path_begin)+1];
+            char hwmon_path[strlen(path_begin)+3];
 
             sprintf(hwmon_path, "%s%d", path_begin, counter);
 
